docs/reference/objects/product/modifier.md

```diff
@@ -0,0 +1,136 @@
+---
+layout: default
+title: Modifier
+parent: Product
+---
+
+When using a `modifier` object you have access to the following attributes:
+
+# modifier.id
+
+Returns a unique id for the modifier.
+
+# modifier.checkbox_attributes
+
+Returns a string of HTML attributes to be used on a checkbox for the modifier.
+
+{% raw %}
+```
+<input {{modifier.checkbox_attributes}}>
+```
+{% endraw %}
+
+# modifier.contents
+
+A list of [biographies]({% link docs/reference/objects/biography.md %}) that are associated with the modifier.
+
+# modifier.custom_fields
+
+A list of custom properties defined on this modifier. Each custom field has two methods: `key` and `value`.
+
+{% raw %}
+```liquid
+<ul>
+  {% for field in modifier.custom_fields %}
+    <li>{{field.key}}: {{field.value }}</li>
+  {% endfor %}
+</ul>
+```
+{% endraw %}
+
+# modifier.custom_fields_data
+
+Returns an object containing the custom properties for the modifier.
+
+{% raw %}
+```
+{{modifier.custom_fields_data.category}}
+```
+{% endraw %}
+
+# modifier.data_attributes
+
+Returns a string of HTML data attributes for the modifier, including `id`, `name` and if present, `start-at` and `end-at`.
+
+{% raw %}
+```
+<input {{modifier.checkbox_attributes}} {{modifier.data_attributes}}>
+```
+{% endraw %}
+
+# modifier.end_at
+
+The end time for this modifier.
+
+{% raw %}
+```liquid
+{% modifier.end_at | date: "%d-%m-%Y %k:%M" %}
+```
+{% endraw %}
+
+# modifier.filter_attributes
+
+Returns a string of HTML data attributes representing the custom field key-value pairs that have been specified as filterable.
+
+For example, a modifier with filterable custom fields and values "category" and "yoga", and "Difficulty" and "hard" would end up with data attributes `data-category="yoga"` and `data-difficulty="hard"`.
```

Comment:
nit: Should we lower case `Difficulty`? I don't think the case is relevant and the inconsistency can create doubts?